nikola/__main__.py

```
@@ -275,13 +275,19 @@ def load_tasks(self, cmd, opt_values, pos_args):
             }
         DOIT_CONFIG['default_tasks'] = ['render_site', 'post_render']
         DOIT_CONFIG.update(self.nikola._doit_config)
-        tasks = generate_tasks(
-            'render_site',
-            self.nikola.gen_tasks('render_site', "Task", 'Group of tasks to render the site.'))
-        latetasks = generate_tasks(
-            'post_render',
-            self.nikola.gen_tasks('post_render', "LateTask", 'Group of tasks to be executed after site is rendered.'))
-        signal('initialized').send(self.nikola)
+        try:
+            tasks = generate_tasks(
+                'render_site',
+                self.nikola.gen_tasks('render_site', "Task", 'Group of tasks to render the site.'))
+            latetasks = generate_tasks(
+                'post_render',
+                self.nikola.gen_tasks('post_render', "LateTask", 'Group of tasks to be executed after site is rendered.'))
+            signal('initialized').send(self.nikola)
+        except Exception:
+            LOGGER.error('Error loading tasks')
+            if self.nikola.debug:
+                raise
+            return {}, {}
```

Perhaps just let the exception bubble upwards and handle it somewhere else.